modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -155,20 +155,30 @@ conf = {{{packageName}}}.Configuration(
 
     _default = None
 
-    def __init__(self, host="{{{basePath}}}",
+    def __init__(self, host=None,
                  api_key=None, api_key_prefix=None,
                  username=None, password=None,
                  discard_unknown_keys=False,
                  disabled_client_side_validations="",
 {{#hasHttpSignatureMethods}}
                  signing_info=None,
 {{/hasHttpSignatureMethods}}
+                 server_index=None, server_variables=None,
+                 server_operation_index=None, server_operation_variables=None,
                  ):
         """Constructor
         """
-        self.host = host
+        self._base_path = "{{{basePath}}}" if host is None else host
```

Comment:
How about naming this variable `_host`, which more clearly conveys that this is the default host that we use?

Reply:
Technically it's a base path since `host` would mean only host part of a URL.